web/src/main/java/com/marklogic/hub/web/service/SearchService.java

```diff
@@ -70,6 +71,15 @@ public StringHandle search(SearchQuery searchQuery) {
 
         ArrayList<StructuredQueryDefinition> queries = new ArrayList<>();
 
+        if (searchQuery.query != null && !searchQuery.query.isNull()) {
```

Comment:
if the condition is true, we will return. Then How to get the line?
https://github.com/marklogic/marklogic-data-hub/blob/8b3813ffeaad6359f866feee4f600a2582b96067/web/src/main/java/com/marklogic/hub/web/service/SearchService.java#L114